ia/src/ml_agent.py

```
@@ -0,0 +1,155 @@
+import protocole
+import game
```

Ruff (0.11.9)

`game` imported but unused

Remove unused import: `game`

(F401)